_episodes/01-run-quit.md

```diff
@@ -5,55 +5,153 @@ exercises: 0
-## Python programs are plain text files.
 
-*   They have the `.py` extension to let everyone (including the operating system) 
-    know it is a Python program.
-    *   This is convention, not a requirement.
-*   It's common to write them using a text editor but we are going to use
-    the [Jupyter Notebook][jupyter].
-*   The bit of extra setup is well worth it because the Notebook provides code completion 
-    and other helpful features.
-*   Notebook files have the extension `.ipynb` to distinguish them from plain-text Python programs.
-    *   Can export as "pure Python" to run from the command line.
+## Getting Started with JupyterLab
 
-## Use the Jupyter Notebook for editing and running Python.
+We will teach Python using [JupyterLab][jupyterlab], a programming environment that runs in a 
+web browser. JupyterLab requires a reasonably up-to-date browser, preferably a current version of 
+Chrome, Safari, or Firefox (note that Internet Explorer version 9 and below are *not* supported).
 
-*   The [Anaconda package manager][anaconda] is an automated way to install the Jupyter notebook.
-    *   See [the setup instructions]({{ page.root }}/setup/) for Anaconda installation instructions.
-*   It also installs all the extra libraries it needs to run.
-*   Once you have installed Python and the Jupyter Notebook requirements, open a shell and type:
+JupyterLab is included as part of the the Anaconda Python distribution. If you have not already 
+installed the Anaconda Python distribution, see [the setup instructions]({{ page.root }}/setup/) 
+for installation instructions.
 
-    ~~~
-    $ jupyter notebook
-    ~~~
+JupyterLab is an application whose frontend user interface runs in a web browser.
 
-*   This will start a Jupyter Notebook server and open your default web browser. 
 *   The server runs locally on your machine only and does not use an internet connection.
-*   The server sends messages to your browser.
-*   The server does the work and the web browser renders the notebook.
-*   You can type code into the browser and see the result when the web page talks to the server.
-*   This has several advantages:
-    *   You can easily type, edit, and copy and paste blocks of code.
-    *   Tab complete allows you to easily access the names of things you are using
-        and learn more about them.
-    *   It allows you to annotate your code with links, different sized text, bullets, etc.
-        to make it more accessible to you and your collaborators.
-    *   It allows you to display figures next to the code that produces them
-        to tell a complete story of the analysis.
+*   The server sends messages to your web browser.
+*   The server does the work and the web browser renders the result.
+*   You will type code into the browser and see the result when the web page talks to the server.
+
+## Starting JupyterLab
+
+### Mac OS X
+To start the JupyterLab server you will need to access the command line through the Terminal. 
```

Comment:
Should we say Terminal app or other command line terminal application like https://www.iterm2.com/ ? Similarly Windows offers PowerShell or even the [WSL](https://medium.com/hugo-ferreiras-blog/using-windows-subsystem-for-linux-for-data-science-9a8e68d7610c)

Reply:
I just pulled this verbiage straight from the `setup.md` file. When I have taught this material the median learner has generally never used a command line terminal application before and thus it seems fine to expose learners to the system defaults. 

Most Windows users, particularly outside US and Western Europe are probably not running Windows 10 (Windows 8 seems to be most widely used here in MENA) so WSL solutions are probably not so useful (unless there are plans to backport?).